docs/01_nodeos/06_logging/index.md

```diff
@@ -105,4 +106,4 @@ Example:
 ```
 
 [[info]]
-| `net_plugin_impl`, `bnet_plugin`, `producer_plugin`, `transaction_tracing` are not enabled unless explicitly enabled in the `logging.json`
+| The default logging level for all loggers if no `logging.json` is provided is `info`. Each logger can be configured independently in the `logging.json` file.
```

Comment:
I would be nice if the doc will explicitly list the possible values for the logging level, right now it is clear two levels are possible: info and debug, but it is not clear if there are any others.

Reply:
They're already listed [here](https://github.com/EOSIO/eos/blob/release/2.0.x/docs/01_nodeos/06_logging/01_logging-levels.md)